### PR TITLE
Bugfix/no breadcrumbs and different font size on mobil

### PR DESCRIPTION
**Pull request by jmikolajczyk**
product_breadcrumbs_mobile
header_static_page_mobile
breadrcrumbs_mobile

### CODE REVIEW

File: src/themes/default/pages/Static.vue [hunk 1]
```diff
@@ -73,6 +73,10 @@ export default {
 @import '~theme/css/helpers/functions/color';
 $border-primary: color(primary, $colors-border);
 
+.static-title {
+  font-size: 36px;
```
Comment (filrak) [inline src/themes/default/pages/Static.vue]: This one should be atomic (fs36)

**Comment by filrak** [comment]
@karolbzik can you review the design part?

File: src/themes/default/pages/Static.vue (hunk 1 above)
Comment (jmikolajczyk) [reply src/themes/default/pages/Static.vue]: Fixed

**Comment by pkarw** [comment]
@jmikolajczyk could you make the changes requested to allow us to merge this PR in?

**Comment by jmikolajczyk** [comment]
@pkarw I added requested changes in this commit https://github.com/DivanteLtd/vue-storefront/pull/1033/commits/f132a9291f39c76ac2df2d7d45dccf3fcd7b79e1

**Pull request merged**